.devcontainer/install-ci-tooling.py

```
@@ -0,0 +1,147 @@
+            [
+                uv_path,
+                "tool",
+                "list",
+            ],
+            check=True,
+            env=uv_env,
+        )
+    if not args.no_node:
+        pnpm_install_sequence = ["npm -v", f"npm install -g pnpm@{PNPM_VERSION}", "pnpm -v"]
+        for cmd in pnpm_install_sequence:
+            cmd = (
```

Ruff (0.12.2)

`for` loop variable `cmd` overwritten by assignment target

(PLW2901)